### PR TITLE
Issue #926 fixed

### LINKED ISSUE #926: Move Account and Profile related code to accounts app
We still have some code in the `api` directory that is related to user accounts/profiles. Move all account/profile code to the `accounts` app and refactor as needed.

Relatedly, rename the `Account` model to `Profile`.

## Task
Below are the overarching steps involved in completing this issue. Some steps are optional but might help us achieve cleaner and more maintainable code.

- [ ] Move all code indicated below to the `accounts` app
- [ ] refactor related code to reference the new location
- [ ] (optional) remove redundant code, if possible
- [ ] (optional) clean-up existing code if there are more conventional ways of doing things

## Related code
All code below should be moved to the `accounts` app.

- [ ] Move `Account` model to the `accounts.models` and rename the model to `Profile`:
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/models/account.py#L1-L3

- [ ] Move Accounts views to the `accounts` app:
- https://github.com/CiviWiki/OpenCiviWiki/blob/develop/project/api/views/account.py

- [ ] Move account/password related forms to `accounts` app:
- https://github.com/CiviWiki/OpenCiviWiki/blob/develop/project/api/forms.py

- [ ] Move User serializer to `accounts` app
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/read.py#L8-L22

- [ ] User card serializer
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/read.py#L25-L40

- [ ] Profile serializer
- https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/read.py#L43-L96

- [ ] Account-related read-only serializers
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/serializers.py#L10-L116

- [ ] Account and profile related URLs
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/urls.py#L10
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/urls.py#L19-L21
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/urls.py#L38-L49

- [ ] user-related utility function
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/utils.py#L12-L22

- [ ] user/profile "write" endpoints
https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/api/write.py#L299-L583

### PR DESCRIPTION
**Pull request by skattel49**
Closes #926 

I have fixed copied the Accounts model to ./accounts/model.py and changed it's name to Profile as instructed

### CODE REVIEW

File: project/accounts/models.py [hunk 1]
```diff
@@ -9,3 +9,135 @@ class User(AbstractUser):
...
+        upload_to=profile_upload_path, blank=True, null=True
+    )
+
+    @property
+    def full_name(self):
+        "Returns the person's full name."
+
+        full_name = "{first_name} {last_name}".format(
+            first_name=self.first_name, last_name=self.last_name
+        )
```
Comment (brylie) [inline project/accounts/models.py]: Let's use f-strings here

```suggestion
        full_name = f"{ self.first_name } { self.last_name }"
```

**Review by brylie — CHANGES_REQUESTED** [review]

**Comment by skattel49** [comment]
I have changed the string formatting to f-strings as suggested.

**Review by brylie — APPROVED** [review]

**Review by brylie — CHANGES_REQUESTED** [review]
This looks good.

Remember to run the command `python manage.py makemigrations` and commit the migrations to this branch.

**Comment by brylie** [comment]
@skattel49, please remember to run `python manage.py makemigrations` and commit the migration file to this branch. Any changes to a model need to have accompanying migration(s).

**Comment by brylie** [comment]
These changes were merged in a separate pull request in order for me to make changes to the branch.

**Pull request closed without merging**